core/src/main/java/io/seata/core/constants/ConfigurationKeys.java

```diff
@@ -106,7 +106,7 @@
     /**
      * The constant CLIENT_REPORT_SUCCESS_ENABLE.
      */
-    public static final String CLIENT_REPORT_SUCCESS_ENABLE = CLIENT_PREFIX + "report.success.enable";
+    public static final String CLIENT_REPORT_SUCCESS_ENABLE = CLIENT_RM_PREFIX + "report.success.enable";
```

Comment:
need to reset this file, `CLIENT_RM_PREFIX = CLIENT_PREFIX + "rm."`  The configuration name here is a dash, not a dot. 
![image](https://user-images.githubusercontent.com/8758457/72050362-e518a880-32fb-11ea-84ad-406805f0948a.png)
![image](https://user-images.githubusercontent.com/8758457/72050391-f661b500-32fb-11ea-913d-a17edf9fc6e5.png)

Reply:
![8385F44E-C62E-40C2-8A37-EA5A97C2AEF3](https://user-images.githubusercontent.com/3851315/72146951-7a896a80-33d8-11ea-9bbf-f58a607b0886.png)